audio/out/ao_coreaudio_chmap.c

```diff
@@ -241,7 +241,9 @@ static AudioChannelLayout* ca_query_stereo_layout(struct ao *ao,
                                                   void *talloc_ctx)
 {
     OSStatus err;
-    const int nch = 2;
+
+#define nch 2
+
```

Comment:
suggestion to avoid VLA and the ugly define:

```c
uint32_t channels[2];
const int nch = MP_ARRAY_SIZE(channels);
```

Reply:
thanks